kite-data/kite-data-core/src/test/java/org/kitesdk/data/spi/filesystem/TestFileSystemWriters.java

```diff
@@ -95,6 +96,43 @@ public void testBasicWrite() throws IOException {
     Assert.assertEquals("Should match written records",
         written, Lists.newArrayList((Iterator) init(reader)));
   }
+  
+  @Test
+  public void testWriteWithOldSchema() throws IOException {
+    Schema writerSchema = SchemaBuilder.record("Message").fields()
```

Comment:
@rdblue  this unit test is now updated to validate the file's schema is same as writerSchema. 
Also A new test has been added to TestRefinableView, where reader writer compatibility is verified by using two schemas (Value, TestValue). 

Let us know if these are the changes you're expecting, or if there are other modifications that needs to be done